src/prefect/blocks/core.py

```diff
@@ -618,6 +623,11 @@ async def load(cls, name: str, client: "OrionClient" = None):
         Args:
             name: The name or slug of the block document. A block document slug is a
                 string with the format <block_type_slug>/<block_document_name>
+            skip_validation: If True, the block document will be loaded without
+                validating the block schema. This is useful if the block schema has
+                changed client-side since the block document referred to by `name` was saved.
```

Comment:
Perhaps I’m lacking the whole picture, but I was wondering, rather than implicitly loading a different version, can we allow this to load a specific version of the block (e.g. replacing skip_validation with version: str)